adafruit_bus_device/i2c_device.py

```diff
@@ -153,18 +153,20 @@ def write_then_readinto(self, out_buffer, in_buffer, *,
                                            out_start=out_start, out_end=out_end,
                                            in_start=in_start, in_end=in_end, stop=stop)
             if self._debug:
-                print("i2c_device.writeto_then_readfrom.out_buffer:", [hex(i) for i in out_buffer])
-                print("i2c_device.writeto_then_readfrom.in_buffer:", [hex(i) for i in in_buffer])
+                print("i2c_device.writeto_then_readfrom.out_buffer:",
+                      [hex(i) for i in out_buffer[out_start:out_end]])
+                print("i2c_device.writeto_then_readfrom.in_buffer:",
```

Comment:
i think print the writebuffer slice *before* the command, then the readbuffer after.